### DISCUSSION TITLE
[linux] Touchscreen redirection

### DISCUSSION THREAD

**Discussion by Blue-Beaker** [LizardByte/community#361] [Sunshine]
I find there is a file "~/.config/sunshine/sunshine_touchscreen" when sunshine is running. so I think there's maybe a way to redirect touchscreen in sunshine at least on linux. but moonlight client doesn't seems like to support this, and just treats touchscreen as an absolute mouse.

And I have a linux tablet with touchscreen, so I tried `ssh user@host cat /dev/input/eventXX > .config/sunshine/sunshine_touchscreen` on host to tredirect touchscreen from the tablet. Then touches is redirected to the host but coords is wrong.

So is there any ways to redirect touchscreen from client to linux host correctly?

**Comment by Boom-Hacker** [comment]
only x11 and wayland WMs can did this
kde gnome etc wayland mode cannot
